### DISCUSSION TITLE
MY DELIVERY IS VERY LATE

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#1891] [Help with using Postal]
How can i get to increase the number of workers please??i checked using POSRAL STATUS and it showed just one worker. Thanks in advance

**Comment by willpower232** [comment]
you need to ask docker to scale up the number of workers which looks like this https://github.com/postalserver/postal/discussions/1461#discussioncomment-1119403

**Reply by Itseric100** [reply]
Thanks so much, it worked and the sending rate is high now but the emails are stuck in PENDING and its too many mails in pending, is there anything i can do to speed that up?
